### ISSUE TITLE
This project is abandoned

### ISSUE THREAD

**Issue by georgeemr** [saribe/eModal#101]
No one is answering all the issues this project has.

**Comment by thomasfrobieter** [comment]
Yes, its not worth using this library in projects >BS3. Basic functionality still works with BS4, but some things (like events) not.
